src/views/settings/Settings.vue

```diff
@@ -15,69 +15,27 @@
         </VForm>
       </VCol>
       <VCol lg="4" md="6" sm="8" cols="12" class="px-3">
-        <VForm
-          v-model="isValid"
-          ref="userForm"
-          @submit.prevent="submitPassword"
-        >
-          <VTextField
-            :label="$t('common.name')"
-            v-model="newUser.name"
-            autocomplete="username"
-            required
-            :rules="[(v) => !!v || $t('errors.user.name-blank')]"
-          />
-          <VTextField
-            :label="$t('users.current-password')"
-            type="password"
-            autocomplete="current-password"
-            v-model="newUser.current_password"
-            :rules="rules.current"
-          />
-          <VTextField
-            :label="$t('users.password')"
-            type="password"
-            autocomplete="new-password"
-            v-model="newUser.password"
-            :rules="rules.password"
-          />
-          <VTextField
-            :label="$t('users.confirm-password')"
-            type="password"
-            autocomplete="new-password"
-            :rules="rules.confirmation"
-            v-model="newUser.password_confirmation"
-          />
-          <VBtn color="primary" class="ma-2" type="submit">
-            {{ $t("common.change-settings") }}
-          </VBtn>
-        </VForm>
+        <UserForm :user="user" redirectFallback="home" />
       </VCol>
     </VRow>
     <VRow>
       <VCol>
-        <AuthTokensTable :authTokens="authTokens" />
+        <AuthTokensTable :authTokens="authTokens" redirectFallback="home" />
```

Comment:
`AuthTokensTable` doesn't take this property (and it shouldn't, since it never redirects).